include/odp/api/spec/ipsec.h

```diff
@@ -71,6 +71,48 @@ typedef enum odp_ipsec_op_mode_t {
 
 } odp_ipsec_op_mode_t;
 
+/**
+ * IPSEC TEST SA parameter type
+ */
+typedef enum odp_ipsec_test_sa_param_type_t {
+	/** Next sequence number value */
+	ODP_IPSEC_TEST_SA_SEQ_NUM = 0,
+
+	/** Highest authenticated sequence number */
+	ODP_IPSEC_TEST_SA_ANTIREPLAY_WINDOW_TOP
+
+} odp_ipsec_test_sa_param_type_t;
```

Comment:
I am not sure if calling this "type" (instead of "instance" or just parameter) is good. This way we cannot use the same "type" for different parameters, suggesting that it is not really a "type" that we are dealing with.

Reply:
Agreed. Updated v5 to use "instance" instead of "type".

Reply:
Agree, this should indicate selection of what the operation does and what fields from the union (under) is used. Command, operation, select ...